reactor-core/src/main/java/reactor/core/scheduler/Scheduler.java

```diff
@@ -63,10 +68,15 @@
 	 * @return the {@link Disposable} that let's one cancel this particular delayed task,
 	 * or throw a {@link RejectedExecutionException} if the Scheduler is not capable of scheduling periodically.
 	 */
+	@Deprecated
```

Comment:
please add a `@deprecated` entry to the javadoc explaining the deprecation and alternative